plugins/thunderbolt/meson.build

```diff
@@ -1,8 +1,15 @@
 cargs = ['-DG_LOG_DOMAIN="FuPluginThunderbolt"']
+tbt_deps = [plugin_deps, gudev]
+
+if get_option('enable-dell')
+  tbt_deps += libsmbios_c
+  tbt_deps += efivar
+endif
 
 shared_module('fu_plugin_thunderbolt',
   sources : [
     'fu-plugin-thunderbolt.c',
+    'fu-dell-common.c',
```

Comment:
I think it's fine to just use the hwid for this. I'm just working on a patchset that allows plugins (like the dell one) set properties on devices from other plugins, e.g. setting a "CanForcePower" metadata value.